src/python/pants/pantsd/process_manager.py

```diff
@@ -488,6 +488,7 @@ def double_fork():
     self.purge_metadata()
     self.pre_fork(**pre_fork_opts or {})
     is_parent, is_child = fork_func()
+    # No arguments reuses the same log location, but with the new pid after forking.
```

Comment:
I think this comment is in the wrong place?

Reply:
Deleted!